openapi/openapi.json

```diff
@@ -3369,5 +3369,15 @@
         "description": "OPTIMADE-specific warning class based on OPTIMADE-specific JSON API Error.\nFrom the specification:\n\n    A warning resource object is defined similarly to a JSON API\n    error object, but MUST also include the field type, which MUST\n    have the value \"warning\". The field detail MUST be present and\n    SHOULD contain a non-critical message, e.g., reporting\n    unrecognized search attributes or deprecated features.\n\nNote: Must be named \"Warnings\", since \"Warning\" is a built-in Python class."
       }
     }
-  }
+  },
+  "servers": [
+    {
+      "url": "https://optimade.herokuapp.com/",
+      "description": "Heroku instance tracking 'master' branch of optimade-python-tools"
+    },
+    {
+      "url": "https://dev-aiida-dev.materialscloud.org/optimade-sample/optimade",
```

Comment:
Are you sure this url:   "https://dev-aiida-dev.materialscloud.org/optimade-sample/optimade" is correct?
I could not connect to it.

Reply:
oh, I just saw this 
![image](https://user-images.githubusercontent.com/1053245/120869813-91b19500-c597-11eb-82a7-58dd4a77d269.png)
I certainly cannot remember ever doing that (sorry if I did this "in my dreams" ;-) )... I still think the idea in this PR makes sense but I currently don't have time to work on it.

The updated URL would be https://aiida.materialscloud.org/optimade-sample/optimade

Reply:
Thank you for providing the correct link.
In May, I started a Postdoc at Cecam and I will be working on the Optimade project. Therefore, Matthew made me code owner for part of the code. Perhaps this triggered the review request. As I am still quite new to the project, I think it would be better to let Matthew and Casper decide about what to do with this pull request.